x-pack/plugins/security_solution/public/flyout/network_details/content.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FC } from 'react';
+import React, { memo } from 'react';
+import { EuiSpacer } from '@elastic/eui';
+import { NetworkDetails } from './components/network_details';
+import { FlyoutBody } from '../shared/components/flyout_body';
+import type { FlowTargetSourceDest } from '../../../common/search_strategy';
+
+export interface PanelContentProps {
+  contextID: string;
```

Comment:
description here?

Reply:
I actually removed this `contextID` entirely as it is always `undefined`